frontend/src/components/annotation/annotation_area_factory.vue

```diff
@@ -55,6 +55,7 @@
     </div>
     <div v-else-if="interface_type === 'audio'">
       <audio_annotation_core
+        v-if="!changing_file && !changing_task && image_annotation_ctx != undefined"
```

Comment:
I had to add this logic for conditional rendering of `audio_annotation_core` because without it `instance_list` is populated inside `audi_annotation_core` before `annotation_ui_context.instance_store` gets reset with the new store instance (which is a sort of a race condition).

Looks like this conditional rendering was originally used as a hack to make saving functionality work for other components (`text_annotation_core`, etc) , since saving logic uses `annotation_ui_context.instance_store` to get instance_list to save